### ISSUE TITLE
URL Bar does not disable 'normal mode'

### ISSUE THREAD

**Issue by Tattomoosa** [akhodakivskiy/VimFx#724]
Text input still registering as commands when focus is on URL bar in Firefox 46.0, Mac OS X 10.11.4.

Interestingly, shifted letters seem to work as they should... j and k scroll but J and K shifted do not switch tabs.

All other text fields seem to work normally for me, including the search bar.

Otherwise great extension but this is a serious issue! If anyone else is having this problem, for the time being you can turn CAPSLOCK on while typing URLs.

**Comment by lydell** [comment]
1. What color is VimFx's toolbar button when the location bar is focused?
2. Could you please test in a clean profile?

**Comment by Tattomoosa** [comment]
The icon greyed out correctly, and it worked properly in a clean profile...

AND upon restarting Firefox back in my profile it seems to working now!

I have gone back to several sites in my history, disabled and re-enabled the only other add-on I have installed (ublock) and cannot seem to reproduce the problem anymore.

Honestly not sure if I had restarted Firefox since I installed VimFx. Appreciate your quick response! Thank you
